scripts/xnat/check_new_sessions

```diff
@@ -192,17 +192,56 @@ def check_dti( ifc, eid, dti_scans ):
 
     return errors
 
+scan_type_dictionary = {'GE MEDICAL SYSTEMS': required_ge.keys(),
+                        'SIEMENS': required_siemens.keys()}
+
 def incomplete_scan_check( eid, manufacturer):
-    scan_id_dictionary = {'GE MEDICAL SYSTEMS': ['1', '10', '11', '12', '13', '3', '4', '6', '7', '8', '9'],
-                            'SIEMENS': ['1', '10', '11', '12', '13', '2', '3', '4', '6', '7', '8', '9']}
-    scan_ids = ifc.select.experiment( eid ).attrs.get('xnat:mrSessionData/scans/scan/id')
+    scan_types = ifc.select.experiment( eid ).attrs.get('xnat:mrSessionData/scans/scan/type')
     if manufacturer == 'GE MEDICAL SYSTEMS':
-        if set(scan_ids) != set(scan_id_dictionary['GE MEDICAL SYSTEMS']):
-            return sibis.logging(eid, 'ERROR: Scans missing from Session')
+        for scantype in set(scan_type_dictionary['GE MEDICAL SYSTEMS']):
+            if scantype in set(scan_types):
+                scantype = scantype
+            else:
+                return sibis.logging(eid, 'ERROR: {} missing from Session'.format(scantype))
     elif manufacturer == 'SIEMENS':
-        if set(scan_ids) != set(scan_id_dictionary['SIEMENS']):
-            return sibis.logging(eid, 'ERROR: Scans missing from Session')
-
+        for scantype in set(scan_type_dictionary['SIEMENS']):
+            if scantype in set(scan_types):
+                if scan_types.count('ncanda-grefieldmap-v1') != 2:
+                    return sibis.logging(eid, 'ERROR: Session does not contain two ncanda-grefieldmap-v1 files')
+            else:
+                return sibis.logging(eid, 'ERROR: {} missing from Session'.format(scantype))
+
+naming_convention_siemens = {'2':['ncanda-t2fse-v1'],
+                            '3': ['ncanda-mprage-v1'],
+                            '4': ['ncanda-dti6b500pepolar-v1'],
+                            '6': ['ncanda-dti60b1000-v1'],
+                            '12': ['ncanda-grefieldmap-v1'],
+                            '13': ['ncanda-grefieldmap-v1'],
+                            '14': ['ncanda-rsfmri-v1']}
+naming_convention_ge = {'3': ['ncanda-t2fse-v1'],
+                        '4': ['ncanda-t1spgr-v1'],
+                        '6': ['ncanda-dti6b500pepolar-v1'],
+                        '7': ['ncanda-dti60b1000-v1'],
+                        '8': ['ncanda-grefieldmap-v1'],
+                        '9': ['ncanda-rsfmri-v1']}
+
+def naming_convention_check(eid, manufacturer):
```

Comment:
@abonil91 sorry for the miscommunication here, we don't need to check that the `scan types` match the specific series number, but good you found out how to grab attributes. Can you cut this chunk and add a commit, then I'll merge?